app/src/main-process/main.ts

```diff
@@ -146,9 +146,27 @@ app.on('ready', () => {
 
   createWindow()
 
-  const menu = buildDefaultMenu()
+  let menu = buildDefaultMenu()
```

Comment:
If we want to have the "Open in XXX" menu item here (and update it whenever it's changed under Preferences, we need to rebuild the menu. Thankfully this wasn't too tough to update and flow through everything.